### PR TITLE
Adding User Count Endpoint

### PR DESCRIPTION
**Pull request by abdullah5abid**

### CODE REVIEW

**Comment by tonytan4ever** [comment]
Can you try fix the failing build ?

File: src/modules/user/user.controller.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  @Get('/count')
+  async count(): Promise<{ count: number }> {
+    try {
+      const count = await this.usersService.count();
+      return {
+        count,
+      };
+    } catch (error) {
+      console.error(error);
```
Comment (tonytan4ever) [inline src/modules/user/user.controller.ts:16]: Remove console error.

**Review by tonytan4ever — CHANGES_REQUESTED** [review]
1. add unit tests.
2. Fix the failing build

**Comment by abdullah5abid** [comment]
> 
I'm trying to resolve the build error but where to add the unit tests?

File: Dockerfile [hunk 2]
```diff
@@ -13,4 +13,4 @@ RUN set -x && yarn run prestart:prod
 
 EXPOSE 3000
 
-CMD [ "node", "-r", "./tsconfig-paths-bootstrap.js" ,"dist/main.js" ]
+CMD [ "node", "-r", "./tsconfig-paths-bootstrap.js" ,"dist/main.js" ]
```
Comment (tonytan4ever) [inline Dockerfile:16]: Can you remove the empty line?

File: docker-compose.yaml [hunk 3]
```diff
@@ -45,8 +45,9 @@ services:
     ports:
       - '4566:4566'
     volumes:
-      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
+      - localstack:/tmp/localstack
       - "/var/run/docker.sock:/var/run/docker.sock"
 
 volumes:
   mysqldata:
+  localstack:
```
Comment (tonytan4ever) [inline docker-compose.yaml:53]: Same here, can you remove the empty line here?

**Review by tonytan4ever — CHANGES_REQUESTED** [review]
Final request:
Can you squash all your 33 commits into one, and fix a few empty lines?
I can merge afterwards.

**Review by tonytan4ever — APPROVED** [review]

**Pull request merged**